src/drivers/dw/dma.c

```diff
@@ -557,9 +557,9 @@ static int dw_dma_set_config(struct dma_chan_data *channel,
 		if (dw_chan->lli)
 			rfree(dw_chan->lli);
 
-		dw_chan->lli = rballoc_align(0, SOF_MEM_CAPS_RAM | SOF_MEM_CAPS_DMA,
-					sizeof(struct dw_lli) * channel->desc_count,
-					PLATFORM_DCACHE_ALIGN);
+		dw_chan->lli = rballoc(SOF_MEM_FLAG_COHERENT,
+				       SOF_MEM_CAPS_RAM | SOF_MEM_CAPS_DMA,
+				       sizeof(struct dw_lli) * channel->desc_count);
```

Comment:
how often is this memory accessed @keyonjie? Is it done for every copy?

Reply:
We access this on every copy for every DMA channel (to clear the block DONE bit). 
@keyonjie I wonder if would could further optimize to only clear the DONE bit of the last 2 LLI elems (rather than the whole list/array). Clearing the last two would reduce uncache IO and allow for drift between timer copies() and DMA block completions.

Reply:
@ranj063 yes it is done for every copy, so typically it is accessed 4 (items) times every 1 millisecond.
@lgirdwood we actually already use uncached address to clear the DONE bit via platform_dw_dma_lli_get(), so I would expect no performance drop with this change applied at all. The only place we are using cached ones is in dw_dma_set_config() with a writeback at the end, but this one is only run one time at the stream starting.

Reply:
ok, that fine then if we are already doing this.